wfl/Models/MenuSelection.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.IdGenerators;
+
+namespace Symphono.Wfl.Models
+{
+    public class MenuSelection: IEntity
+    {
+        [BsonIgnoreIfNull]
+        [BsonId(IdGenerator = typeof(StringObjectIdGenerator))]
+        public string Id { get; set; }
+        public string OrdererName { get; set; }
+        public string Description { get; set; }
+        public string FoodOrderId { get; set; }
```

Comment:
Makes sense, but maybe it would be better to instead store an enumerable of menu selection objects inside of a food order.